tools/pubsys/src/main.rs

```diff
@@ -39,8 +40,17 @@ fn run() -> Result<()> {
     // Parse and store the args passed to the program
     let args = Args::from_args();
 
-    // SimpleLogger will send errors to stderr and anything less to stdout.
-    SimpleLogger::init(args.log_level, LogConfig::default()).context(error::LoggerSnafu)?;
+    match args.log_level {
+        // Set log level for AWS SDK to error to reduce verbosity.
+        LevelFilter::Info => Builder::new()
+            .filter_level(args.log_level)
+            .filter(Some("aws_config"), LevelFilter::Error)
+            .filter(Some("aws_smithy"), LevelFilter::Error)
```

Comment:
What does filtering at `Warning` look like for the `aws-sdk-rust` crates? Is it just as noisy?

Reply:
I set to filter at `Error` since that's the level that was set in `bottlerocket-test-system`, but `Warn` could probably make sense here.